api/database_cluster_restore.go

```diff
@@ -34,6 +37,8 @@ const (
 )
 
 // ListDatabaseClusterRestores List of the created database cluster restores on the specified kubernetes cluster.
+//
+//nolint:dupl
```

Comment:
Is it possible to tackle this linter?

Reply:
We could abstract the functionality, although I think it will be harder to read. The code was duplicated before this change as well.
I understand, since we use `nolint:dupl` in other places, this is a good tradeoff between abstraction and readability.

If you look at the two functions, there would be 4-5 parameters passed in, most probably we would pass around interfaces, would need type assertion, etc. Not sure if it's adding any value tbh.